test/oas3/execute/main.js

```diff
@@ -665,6 +665,23 @@ describe('buildRequest - OpenAPI Specification 3.0', () => {
 
       expect(res).toEqual('https://gist.githubusercontent.com');
     });
+    it('should parse relative server url against contextUrls correctly', () => {
```

Comment:
Can we have the table you created as PR comment  translated here into actual tests?

Reply:
Done. Though I'm not sure whether the coding style is acceptable.

Reply:
Coding style is fine. Changed some nitpicks myself.